crates/interpreter/src/side_table.rs

```diff
@@ -43,24 +43,28 @@ impl<'m> Metadata<'m> {
         self.0[0] as usize
     }
 
+    pub fn code_start(&self) -> usize {
```

Comment:
This shouldn't be store in each function, aka `Metadata`, it should be in `SideTable` (before `indices`).